src/avt/Math/CMakeLists.txt

```diff
@@ -7,34 +7,30 @@
 #
 #****************************************************************************
 
-SET(AVTMATH_SOURCES
-avtFactorial.C
-avtHermitePolynomial.C
-avtMatrix.C
-avtPlane.C
-avtPolynomial.C
-avtQuaternion.C
-avtTrackball.C
-avtVecArray.C
-avtVector.C
-)
-
-INCLUDE_DIRECTORIES(
-${CMAKE_CURRENT_SOURCE_DIR}
-${VISIT_COMMON_INCLUDES}
-${VTK_INCLUDE_DIRS}
-)
-
 if(VTK_VERSION VERSION_EQUAL "8.1.0")
     set(vtklibs vtkCommonCore)
 else()
     set(vtklibs VTK::CommonCore)
 endif()
 
-# Add link directories
-LINK_DIRECTORIES(${LIBRARY_OUTPUT_DIRECTORY} )
+blt_add_library(
```

Comment:
So, this is a tad different than prev. example. This is a `blt_add_library` happening down in a *leaf* subdirectory. So, just list the sources and includes explicitly there because that list isn't getting shared or extended by any other subdirectories.

Reply:
correct